AM.md

```diff
@@ -0,0 +1,203 @@
+
+            * Name:  Scripted Module - Client Side Example
+            * Script Type: Client-side Authentication
+
+        1. Select the Create button.
+
+            In the next dialog, with the new script properties, populate the Script input with the following JavaScript code:
+
+            * Script:
+
+            ```javascript
+            /*
+            * Object to hold the data to be passed to the server-side processing.
+            *
+            * The name of this object can be any valid variable name.
+            */
+            var data = {};
+
+            /*
+            * Script element for loading an external library.
+            *
+            * When the script is loaded, it will make a request to an external source to obtain the client's IP information.
+            *
+            * The information, received as a JSON object, is then saved as a string in the
+            * value of the `output` input in the form provided on the client side automatically. The value will become available to the server-side script as the `clientScriptOutputData` javascript variable.
+            */
+            var script = document.createElement('script');
+
+            script.src = 'https://code.jquery.com/jquery-3.4.1.min.js';
+            script.onload = function (e) {
+                $.getJSON('https://ipgeolocation.com/?json=1', function (json) {
+                clientScriptOutputData.ip = json;
+                    output.value = JSON.stringify(data);
+
+                    submit();
+                });
+            }
+
+            document.getElementsByTagName('head')[0].appendChild(script);
+            /*
+            *
+            * To allow for the asynchronous script operation to complete,
+            * automatic submission of the form on the page is delayed via a setting that takes milliseconds.
+            */
+            autoSubmitDelay = 4000;
+            ```
+
+            The language for a client-side script is always JavaScript, for the script's run time environment is assumed to be a browser of some sort.
+
+        1. Select the Save Changes button.
+
+        The expected result returned from the call to `https://ipgeolocation.com/?json=1` is a JSON containing client IP data. The data might look like the following:
+
+        ```json
+        {
+            "ip": "73.67.228.195",
+            "city": "Portland",
+            "region": "Oregon",
+            "country": "United States",
+            "coords": "45.547200,-122.641700",
+            "asn": "AS7922, Comcast Cable Communications, LLC",
+            "postal": "97212",
+            "timezone": "America/Los_Angeles"
+        }
+        ```
+
+        The script uses stringified form of this data to populate the `output` input in the form on the page where the client-side script is running. When the form is submitted, the value of the input will become available for the server-side scripts as the `clientScriptOutputData` variable.
```

Comment:
So `clientScriptOutputData` is a special variable name for capturing the result from the script?

Reply:
Yes, in a-chains. (People on market place continue to use it in custom nodes—for consistency sake, I assume.)